docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
+  express-server:
+    build: .
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}/${IMAGE_VERSION}
+    env_file: .env
```

Comment:
Don't need `env_file` here. Usually this file just specifies build context and image namespace. Environment will be set in `docker-compose.override.yml` or injected at runtime in a production deployment.